Makefile

```diff
@@ -41,6 +42,9 @@ build-test-wasms:
 
 build-test: build-test-wasms install
 
+generate-full-help-doc:
+	cargo run --bin doc-gen --features clap-markdown
```

Comment:
For some reason this one target was built not with a makefile target, but I moved it here for consistency.